Tools/CISupport/delete-stale-build-files

```diff
@@ -86,6 +91,20 @@ def webkitBuildDirectory(platform, fullPlatform, configuration):
     return subprocess.Popen(['perl', os.path.join(os.path.dirname(__file__), "..", "Scripts", "webkit-build-directory"),
         "--" + platform, "--" + configuration, '--top-level'], stdout=subprocess.PIPE).communicate()[0].strip()
 
+def deleteWindowsStaleFiles():
+    directoriesToDelete = ['/cygdrive/c/Program Files (x86)/Windows Kits/10/Debuggers/x64/sym/WebKit.pdb',
+                           '/cygdrive/c/Program Files (x86)/Windows Kits/10/Debuggers/x64/sym/WTF.pdb',
+                           '/cygdrive/c/Program Files (x86)/Windows Kits/10/Debuggers/x64/sym/JavaScriptCore.pdb',
+                           '/cygdrive/c/Program Files (x86)/Windows Kits/10/Debuggers/x64/sym/DumpRenderTreeLib.pdb']
+    for directory in directoriesToDelete:
+        try:
+            print(f'Removing: {directory}')
+            shutil.rmtree(directory)
+        except OSError as e:
+            print(f'Failed to remove: {directory}, error: {e}')
+            continue
+        print('Done')
```

Comment:
do you want to print Done every single time?

Reply:
It doesn't look ugly as such: https://ews-build.webkit-uat.org/#/builders/13/builds/4316/steps/5/logs/stdio, but I can remove if anyone has a strong opinion on it.